webcompat/api/endpoints.py

```diff
@@ -105,8 +108,12 @@ def get_user_activity_issues(username, parameter):
     # copy the params so we can add to the dict.
     params = request.args.copy()
     params['state'] = 'all'
-    params['{0}'.format(parameter)] = '{0}'.format(username)
+    if parameter == 'needsinfo':
+        params['labels'] = 'status-needsinfo-{0}'.format(username)
+    else:
+        params['{0}'.format(parameter)] = '{0}'.format(username)
```

Comment:
I first wonder about why `'{0}'.format(parameter)` instead of `parameter`, then I though wait this is the same thing for username. (not your code which is fine, but the previous version.)

```python
>>> params = {}
>>> parameter = u'created'
>>> type(parameter)
<type 'unicode'>
>>> username = u'miketaylr'
>>> type(username)
<type 'unicode'>
>>> params[parameter] = username
>>> params
{u'created': u'miketaylr'}
>>> parameter.encode('utf-8')
'created'
>>> params[parameter.encode('utf-8')] = username.encode('utf-8')
>>> params
{u'created': 'miketaylr'}
>>> parameter = parameter.encode('utf-8')
>>> parameter
'created'
>>> params[parameter] = username.encode('utf-8')
>>> params
{u'created': 'miketaylr'}
>>> params['created']
'miketaylr'
```

Reply:
yeah, good question. it's possible it was a more complex formatting during some "in progress" version, and then was trimmed without much thought. :)

let's fix here.